projects/ion/src/lib/info-badge/info-badge.component.ts

```diff
@@ -22,4 +22,12 @@ export class InfoBadgeComponent {
   @Input() public size?: InfoBadgeSize = 'md';
 
   public iconSize: IconSizeOptions = IconSizeOptions[this.size];
+
+  fixIconSize() {
+    if (this.icon === 'exclamation' || this.icon === 'info') {
+      return this.size === 'md' ? (this.iconSize = 20) : (this.iconSize = 14);
```

Comment:
I think it would be better to create an enum with the sizes and return something like this.iconSize = sizesEnum[this.size].